src/ServiceControl.sln

```diff
@@ -157,8 +157,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ServiceControl.Transports.M
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ServiceControl.RavenDB", "ServiceControl.RavenDB\ServiceControl.RavenDB.csproj", "{D8610AB6-EB24-4617-8089-F11732D3E033}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Particular.LicensingComponent", "Particular.LicensingComponent", "{2D732554-9C3D-4509-8D0B-5595266A4D92}"
```

Comment:
I assume this just adds a folder to the solution, not really a project, correct?